### DISCUSSION TITLE
part-ii/2018-23/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#137] [Question comments]
Part II 2018 Automata and Formal Languages: Paper 3, Section I, G 

https://questions.tripos.org/part-ii/2018-23/

**Comment by artie2000** [comment]
(c): I get $A\to a,\ B\to b,\ C\to c,\ S\to AD\mid AE,\ D\to BC,\ E\to SC$.
